### ISSUE TITLE
Add central SCC size as metadata item

### ISSUE THREAD

**Issue by daemontus** [sybila/biodivine-boolean-models-website#21]
This is mostly a task for @daemontus, but it would be nice to have the size of the "central" SCC as part of the metadata (or all non-trivial SCCs). This is what typically really distinguishes "hard" networks from the "easy" ones, not the variable count.

**Comment by AurumTheEnd** [comment]
From my understanding, this issue has now changed to utilizing the metadata which @lchudicek has implemented for the models.

**Comment by daemontus** [comment]
Indeed, this would basically mean "something needs to be added to metadata and then the website will need to read that metadata"
